<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupDialog</class>
 <widget class="QDialog" name="SetupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="btn_open">
     <property name="text">
      <string>Open Game</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Player Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edit_player_name">
       <property name="text">
        <string>Player</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Number of Bots</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spin_num_bots">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>5</number>
       </property>
       <property name="value">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="grp_difficulty">
     <property name="title">
      <string>Bot Difficulty</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="radio_easy">
        <property name="text">
         <string>Ea&amp;sy</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radio_medium">
        <property name="text">
         <string>&amp;Medium</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grp_bot_names">
     <property name="title">
      <string>Bot Names</string>
     </property>
     <layout class="QVBoxLayout" name="lyout_bot_names">
      <item>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="text">
         <string>Bot1</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="btn_play">
     <property name="text">
      <string>Play Go Fish</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edit_player_name</tabstop>
  <tabstop>spin_num_bots</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>btn_play</tabstop>
  <tabstop>btn_open</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
